src/OpenTelemetry/Trace/Export/SimpleSpanProcessor.cs

```diff
@@ -37,14 +45,28 @@ public override void OnStart(Span span)
         /// <inheritdoc />
         public override void OnEnd(Span span)
         {
-            // do not await, just start export
-            this.Exporter.ExportAsync(new[] { span }, CancellationToken.None);
+            try
+            {
+                // do not await, just start export
+                // it can still throw in synchronous part
+                this.Exporter.ExportAsync(new[] { span }, CancellationToken.None);
```

Comment:
Is this missing a `ContinuesWith` or something to observe exceptions? Wouldn't it end up on `TaskUnobservedException` otherwise?

Reply:
been thinking about it: let's say we do `ContinueWith` and log on error. This will be extra task allocation on each span with the only reason to log about malfunctioning exporter (exporter should not throw and should return a result to notify about errors).
Considering that malfunctioning exporters are rare and ContinueWith will make everyone who uses simple processor little miserable all the time, I'd say we should not do it until we have a strong reason to.

We cannot await here - it's non-blocking `OnEnd` call and queueing will make his processor a variation of Batching exporter. So I guess there is nothing reasonable we can do here.

All this gets me back to the question - why simple processor even exists.